_src/pldoc/static/sass/main-ltr.scss

```diff
@@ -6,17 +6,9 @@
 // NOTE: This is the left to right (LTR) configured style compile. It should mirror main-rtl w/ the exception of bi-app references.
 
 // ------------------------------
-// #VENDOR
+// #CONFIG - layout direction
```

Comment:
Not in this section, but in the ``build.scss`` file, there is a place for the app's ``_config.scss`` which can add/replace variables, functions, mixins.